lms/djangoapps/instructor_task/api.py

```diff
@@ -45,6 +45,11 @@
 from util import milestones_helpers
 
 
+
+class SpecificStudentIdMissingError(Exception):
+    """Exception indicating that a background task is already running"""
```

Comment:
doc string is inaccurate :)
"""
Exception indicating that a student id was not provided when generating a certificate for a specific student.
"""